src/styles/utils/generators/TooltipStyleUtils/index.ts

```diff
@@ -269,6 +262,20 @@ const createTooltipStyleUtils: StyleUtilGenerator<GetTooltipStylesStyleUtil> = (
             },
         };
     },
+
+    /** Utility function to create and manage scale animations with React Native Reanimated */
+    getTooltipAnimatedStyles: (props: TooltipAnimationProps) => {
+        const tooltipHorizontalPadding = spacing.ph2.paddingHorizontal * 2;
+        const tooltipWidth = props.tooltipContentWidth && props.tooltipContentWidth + tooltipHorizontalPadding + 1;
+        const isTooltipSizeReady = tooltipWidth !== undefined && props.tooltipWrapperHeight !== undefined;
+        let scale = 1;
```

Comment:
✋ Hey, coming from https://github.com/Expensify/App/issues/55848#issuecomment-2676489115
This has caused an issue with tooltip flickering before showing fully

`scale` here defaults to 1 when `isTooltipSizeReady` is false, which displays the tooltip even before it's size was calculated. This was resolved by changing this line to `let scale = 0`, so the tooltip is hidden until the size is calculated
More details: https://github.com/Expensify/App/issues/55848#issuecomment-2676489115